pelican/__init__.py

```diff
@@ -330,10 +332,12 @@ def get_config(args):
         config['RELATIVE_URLS'] = args.relative_paths
     config['DEBUG'] = args.verbosity == logging.DEBUG
 
-    # argparse returns bytes in Py2. There is no definite answer as to which
-    # encoding argparse (or sys.argv) uses.
-    # "Best" option seems to be locale.getpreferredencoding()
-    # ref: http://mail.python.org/pipermail/python-list/2006-October/405766.html
+    """
```

Comment:
What is the benefit of changing these comments to a docstring? It's not actually documenting anything.